src/frontend/src/handler/query.rs

```diff
@@ -386,9 +386,7 @@ async fn execute(
                         "no affected rows in output".to_string(),
                     )))
                 }
-                Some(row) => {
-                    row.map_err(|err| RwError::from(ErrorCode::InternalError(format!("{}", err))))?
```

Comment:
Here's the reason. Directly formatting on the `err: BoxedError` loses the info of the source.